packages/navigator/src/browser/navigator-preferences.ts

```diff
@@ -24,12 +24,18 @@ export const FileNavigatorConfigSchema: PreferenceSchema = {
             type: 'boolean',
             description: 'Selects file under editing in the explorer.',
             default: true
-        }
+        },
+        'explorer.countBadge': {
```

Comment:
Can we align the naming with that present in `vscode`?

```
explorer.decorations.badges
```

Aligning means that when we read `.vscode/settings.json` preferences (which is supported by the framework), it'll work out of the box. Other preferences should follow this pattern (ex: `scm.decorations.badges`).